Template/Trigger/CustomEventTrigger.php

```diff
@@ -29,8 +30,8 @@ public function getParameters()
     {
         return array(
             $this->makeSetting('eventName', '', FieldConfig::TYPE_STRING, function (FieldConfig $field) {
-                $field->title = 'Event Name';
-                $field->description = 'The name of the event that is pushed to the Data-Layer. For example you can push an event by adding this to your website: _mtm.push({"event": "my-custom-event"});';
+                $field->title = Piwik::translate('TagManager_CustomEventTriggerEventNameTitle');
```

Comment:
It looks like `Events_EventName` already exists and has the same value. The Event plugin appears to be part of core, so I'm guessing it's safe to use in another plugin.

Reply:
@snake14 Not sure if we should use that

Reply:
updated as suggested